src/data/contributors/kunal.json

```diff
@@ -1,6 +1,6 @@
 {
-  "name": "Kunal Kushwaha",
-  "skills": ["ML", "DevOps", "FullStack"],
+  "name": "lucky",
+  "skills": ["ML", "web development", "FullStack"],
```

Comment:
hey @luckykumarirai, You should not change the already existed database.
Instead you should create your own seperate `json` file with your name and start writing on it.
Here is a sample PR you can take a look at: https://github.com/codeforcauseorg/pledge/pull/106/files

Also you should provide your own github/linkedIn/twitter handles. If not you can also omit some.